webapp/channels/src/components/autocomplete_selector.tsx

```diff
@@ -101,7 +123,7 @@ export default class AutocompleteSelector extends React.PureComponent {
         const {focused} = this.state;
         let {input} = this.state;
 
-        if (!focused) {
+        if (!focused && value !== undefined) {
```

Comment:
`value` is going to be always different to undefined, because it is set with the defaultProps. Hopefully updating the props as I said in my last comment will make this change unneeded.